Makefile

```diff
@@ -2,8 +2,10 @@
 
 SHELL = /bin/sh
 
-CURRENT_UID := $(shell id -u)
-CURRENT_UID := $(shell id -g)
+ifndef mac
```

Comment:
@submarcos That's the cleanest way I found to do it.

On mac, the commands should be called like this `make build_admin mac=true` or `make build_front mac=1`